internal/tools/go.mod

```diff
@@ -245,7 +245,7 @@ require (
 	go.opencensus.io v0.24.0 // indirect
 	go.uber.org/atomic v1.9.0 // indirect
 	go.uber.org/fx v1.13.1 // indirect
-	go.uber.org/multierr v1.8.0 // indirect
+	go.uber.org/multierr v1.11.0 // indirect
```

Comment:
I should check what exactly this changes.  I doubt it matters to our code now, but it could in the future.

Reply:
yea, looks fine.  cleanups / new top-level funcs, all seem reasonable.